StoreApp.BLL/Services/ProducerService.cs

```diff
@@ -16,13 +16,15 @@ public class ProducerService : IProducerService
 
         public ProducerService(IUnitOfWork uof)
         {
-            config = new MapperConfiguration(cfg => cfg.CreateMap<Producer, ProducerDTO>()).CreateMapper();
+            config = new MapperConfiguration(cfg => { cfg.CreateMap<Producer, ProducerDTO>();
```

Comment:
Btw if we keep this code here we should format it

Reply:
fixed